### DISCUSSION TITLE
Brownie does not display private keys

### DISCUSSION THREAD

**Discussion by praxis-dev** [smartcontractkit/full-blockchain-solidity-course-py#1492] [Q&A] [answered]
Hi!

I've stored private keys in brownie natively through 

`brownie accounts new ...` 

when i request accounts list or run the deploy.py as directed in the video on 4:37 it shows only addresses, but not private keys.

Same results with deploy.py execution.

Could someone comment on that?

**Comment by cromewar** [comment, marked as answer]
That's the point on related to the private keys they are encrypted on brownie in order to avoid security risks. The private keys must be hidden all times.
